examples/util/WebNNRunner.js

```diff
@@ -0,0 +1,427 @@
+class WebNNRunner extends BaseRunner {
+  constructor() {
+    super();
+    this._currentBackend = null;
+    this._currentPrefer = null;
+    this._outputTensor = [];
+    this._rawModel = null;
+    this._subgraphsSummary = [];
+    this._modelRequiredOps = null;
+    this._deQuantizeParams = null;
+    this._bEagerMode = false;
+    this._supportedOps = new Set();
+  }
+
+  /**
+   * This method is to set '_currentBackend'.
+   * @param backend: A string for backend.
+   */
+  _setBackend = (backend) => {
+    this._currentBackend = backend;
+  };
+
+  /**
+   * This method is to set '_currentPrefer'.
+   * @param prefer: A string for prefer.
+   */
+  _setPrefer = (prefer) => {
+    this._currentPrefer = prefer;
+  };
+
+  /**
+   * This method is to set '_bEagerMode'.
+   * @param flag: A boolean whether eager mode.
+   */
+  setEagerMode = (flag) => {
+    this._bEagerMode = flag;
+  };
+
+  /**
+   * This method is to set '_supportedOps'.
+   * @param ops: A string for supported ops.
+   */
+  setSupportedOps = (ops) => {
+    this._supportedOps = ops;
+  };
+
+  /**
+   * This method is to set '_rawModel'.
+   * @param model: A string for model.
+   */
+  _setRawModel = (model) => {
+    this._rawModel = model;
+  };
+
+  /**
+   * This method is to set '_subgraphsSummary'.
+   * @param summary: A string for summary.
+   */
+  _setSubgraphsSummary = (summary) => {
+    this._subgraphsSummary = summary;
+  };
+
+  /**
+   * This method is to set '_modelRequiredOps'.
+   * @param ops: A string for required ops of model.
+   */
+  _setModelRequiredOps = (ops) => {
+    this._modelRequiredOps = ops;
+  };
+
+  /**
+   * This method is to set '_deQuantizeParams'.
+   * @param params: A string for deQuantize params.
+   */
+  _setDeQuantizeParams = (params) => {
+    this._deQuantizeParams = params;
+  };
+
+  /**
+   * This method is to load model file with specified url.
+   * @param url: A string for model file url.
+   * @returns {string} This returns status of load model file, ['ERROR' | 'SUCCESS'].
```

Comment:
Does this API really need to return a status string? Does the caller check that? If there is a failure, it throws. If no specific usage of the returning value, I would suggest not to return.